java.lang.NoClassDefFoundError: com/sun/jna/Platform

```
$ sbt publishM2 && cd plugin-tester-java && ./gradlew --include-build ../gradle-plugin clean compileJava
Detected sbt version 1.1.1
Downloading sbt launcher for 1.1.1:
  From  http://repo.scala-sbt.org/scalasbt/maven-releases/org/scala-sbt/sbt-launch/1.1.1/sbt-launch.jar
    To  /home/travis/.sbt/launchers/1.1.1/sbt-launch.jar
Using jvm options defined in file /etc/sbt/jvmopts
# Executing command line:
java
-Xms2048M
-Xmx2048M
-Xss6M
-XX:MaxPermSize=512M
-jar
/home/travis/.sbt/launchers/1.1.1/sbt-launch.jar
publishM2
saving stty: 500:5:bf:8a3b:3:1c:7f:15:4:0:1:0:11:13:1a:0:12:f:17:16:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=512M; support was removed in 8.0
Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m
[error] java.lang.NoClassDefFoundError: com/sun/jna/Platform
[error] 	at sbt.internal.io.Milli$.<init>(Milli.scala:364)
[error] 	at sbt.internal.io.Milli$.<clinit>(Milli.scala)
[error] 	at sbt.io.IO$.getModifiedTime(IO.scala:1143)
[error] 	at sbt.io.IO$.getModifiedTimeOrZero(IO.scala:1212)
[error] 	at sbt.compiler.Eval$.fileModifiedBytes(Eval.scala:491)
[error] 	at sbt.compiler.Eval.$anonfun$evalCommon$2(Eval.scala:165)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
[error] 	at scala.collection.immutable.List.map(List.scala:295)
[error] 	at sbt.compiler.Eval$.seqBytes(Eval.scala:482)
[error] 	at sbt.compiler.Eval.evalCommon(Eval.scala:165)
[error] 	at sbt.compiler.Eval.eval(Eval.scala:119)
[error] 	at sbt.internal.EvaluateConfigurations$.evaluateDslEntry(EvaluateConfigurations.scala:180)
[error] 	at sbt.internal.EvaluateConfigurations$.$anonfun$evaluateSbtFile$2(EvaluateConfigurations.scala:120)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
[error] 	at scala.collection.immutable.List.map(List.scala:295)
[error] 	at sbt.internal.EvaluateConfigurations$.evaluateSbtFile(EvaluateConfigurations.scala:118)
[error] 	at sbt.internal.Load$.loadSettingsFile$1(Load.scala:1087)
[error] 	at sbt.internal.Load$.$anonfun$discoverProjects$2(Load.scala:1095)
[error] 	at scala.collection.MapLike.getOrElse(MapLike.scala:128)
[error] 	at scala.collection.MapLike.getOrElse$(MapLike.scala:126)
[error] 	at scala.collection.AbstractMap.getOrElse(Map.scala:59)
[error] 	at sbt.internal.Load$.memoLoadSettingsFile$1(Load.scala:1094)
[error] 	at sbt.internal.Load$.$anonfun$discoverProjects$4(Load.scala:1102)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
[error] 	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:59)
[error] 	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:52)
[error] 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
[error] 	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
[error] 	at sbt.internal.Load$.loadFiles$1(Load.scala:1102)
[error] 	at sbt.internal.Load$.discoverProjects(Load.scala:1116)
[error] 	at sbt.internal.Load$.discover$1(Load.scala:844)
[error] 	at sbt.internal.Load$.loadTransitive(Load.scala:909)
[error] 	at sbt.internal.Load$.loadProjects$1(Load.scala:714)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$10(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$1(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.loadUnit(Load.scala:677)
[error] 	at sbt.internal.Load$.$anonfun$builtinLoader$4(Load.scala:477)
[error] 	at sbt.internal.BuildLoader$.$anonfun$componentLoader$5(BuildLoader.scala:158)
[error] 	at sbt.internal.BuildLoader.apply(BuildLoader.scala:223)
[error] 	at sbt.internal.Load$.loadURI$1(Load.scala:539)
[error] 	at sbt.internal.Load$.loadAll(Load.scala:555)
[error] 	at sbt.internal.Load$.loadURI(Load.scala:485)
[error] 	at sbt.internal.Load$.load(Load.scala:464)
[error] 	at sbt.internal.Load$.$anonfun$apply$1(Load.scala:253)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.apply(Load.scala:253)
[error] 	at sbt.internal.Load$.buildPluginDefinition(Load.scala:1261)
[error] 	at sbt.internal.Load$.buildPlugins(Load.scala:1191)
[error] 	at sbt.internal.Load$.plugins(Load.scala:1174)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$2(Load.scala:683)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$1(Load.scala:683)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.loadUnit(Load.scala:677)
[error] 	at sbt.internal.Load$.$anonfun$builtinLoader$4(Load.scala:477)
[error] 	at sbt.internal.BuildLoader$.$anonfun$componentLoader$5(BuildLoader.scala:158)
[error] 	at sbt.internal.BuildLoader.apply(BuildLoader.scala:223)
[error] 	at sbt.internal.Load$.loadURI$1(Load.scala:539)
[error] 	at sbt.internal.Load$.loadAll(Load.scala:555)
[error] 	at sbt.internal.Load$.loadURI(Load.scala:485)
[error] 	at sbt.internal.Load$.load(Load.scala:464)
[error] 	at sbt.internal.Load$.$anonfun$apply$1(Load.scala:253)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.apply(Load.scala:253)
[error] 	at sbt.internal.Load$.defaultLoad(Load.scala:69)
[error] 	at sbt.BuiltinCommands$.liftedTree1$1(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.doLoadProject(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.$anonfun$loadProjectImpl$2(Main.scala:680)
[error] 	at sbt.Command$.$anonfun$applyEffect$4(Command.scala:134)
[error] 	at sbt.Command$.$anonfun$applyEffect$2(Command.scala:130)
[error] 	at sbt.MainLoop$.processCommand(MainLoop.scala:153)
[error] 	at sbt.MainLoop$.$anonfun$next$2(MainLoop.scala:136)
[error] 	at sbt.State$$anon$1.runCmd$1(State.scala:242)
[error] 	at sbt.State$$anon$1.process(State.scala:248)
[error] 	at sbt.MainLoop$.$anonfun$next$1(MainLoop.scala:136)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.MainLoop$.next(MainLoop.scala:136)
[error] 	at sbt.MainLoop$.run(MainLoop.scala:129)
[error] 	at sbt.MainLoop$.$anonfun$runWithNewLog$1(MainLoop.scala:107)
[error] 	at sbt.io.Using.apply(Using.scala:22)
[error] 	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:101)
[error] 	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:57)
[error] 	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:42)
[error] 	at sbt.MainLoop$.runLogged(MainLoop.scala:34)
[error] 	at sbt.StandardMain$.runManaged(Main.scala:113)
[error] 	at sbt.xMain.run(Main.scala:76)
[error] 	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:109)
[error] 	at xsbt.boot.Launch$.withContextLoader(Launch.scala:128)
[error] 	at xsbt.boot.Launch$.run(Launch.scala:109)
[error] 	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
[error] 	at xsbt.boot.Launch$.launch(Launch.scala:117)
[error] 	at xsbt.boot.Launch$.apply(Launch.scala:18)
[error] 	at xsbt.boot.Boot$.runImpl(Boot.scala:41)
[error] 	at xsbt.boot.Boot$.main(Boot.scala:17)
[error] 	at xsbt.boot.Boot.main(Boot.scala)
[error] Caused by: java.lang.ClassNotFoundException: com.sun.jna.Platform
[error] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[error] 	at sbt.internal.io.Milli$.<init>(Milli.scala:364)
[error] 	at sbt.internal.io.Milli$.<clinit>(Milli.scala)
[error] 	at sbt.io.IO$.getModifiedTime(IO.scala:1143)
[error] 	at sbt.io.IO$.getModifiedTimeOrZero(IO.scala:1212)
[error] 	at sbt.compiler.Eval$.fileModifiedBytes(Eval.scala:491)
[error] 	at sbt.compiler.Eval.$anonfun$evalCommon$2(Eval.scala:165)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
[error] 	at scala.collection.immutable.List.map(List.scala:295)
[error] 	at sbt.compiler.Eval$.seqBytes(Eval.scala:482)
[error] 	at sbt.compiler.Eval.evalCommon(Eval.scala:165)
[error] 	at sbt.compiler.Eval.eval(Eval.scala:119)
[error] 	at sbt.internal.EvaluateConfigurations$.evaluateDslEntry(EvaluateConfigurations.scala:180)
[error] 	at sbt.internal.EvaluateConfigurations$.$anonfun$evaluateSbtFile$2(EvaluateConfigurations.scala:120)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
[error] 	at scala.collection.immutable.List.map(List.scala:295)
[error] 	at sbt.internal.EvaluateConfigurations$.evaluateSbtFile(EvaluateConfigurations.scala:118)
[error] 	at sbt.internal.Load$.loadSettingsFile$1(Load.scala:1087)
[error] 	at sbt.internal.Load$.$anonfun$discoverProjects$2(Load.scala:1095)
[error] 	at scala.collection.MapLike.getOrElse(MapLike.scala:128)
[error] 	at scala.collection.MapLike.getOrElse$(MapLike.scala:126)
[error] 	at scala.collection.AbstractMap.getOrElse(Map.scala:59)
[error] 	at sbt.internal.Load$.memoLoadSettingsFile$1(Load.scala:1094)
[error] 	at sbt.internal.Load$.$anonfun$discoverProjects$4(Load.scala:1102)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
[error] 	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:59)
[error] 	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:52)
[error] 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
[error] 	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
[error] 	at sbt.internal.Load$.loadFiles$1(Load.scala:1102)
[error] 	at sbt.internal.Load$.discoverProjects(Load.scala:1116)
[error] 	at sbt.internal.Load$.discover$1(Load.scala:844)
[error] 	at sbt.internal.Load$.loadTransitive(Load.scala:909)
[error] 	at sbt.internal.Load$.loadProjects$1(Load.scala:714)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$10(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$1(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.loadUnit(Load.scala:677)
[error] 	at sbt.internal.Load$.$anonfun$builtinLoader$4(Load.scala:477)
[error] 	at sbt.internal.BuildLoader$.$anonfun$componentLoader$5(BuildLoader.scala:158)
[error] 	at sbt.internal.BuildLoader.apply(BuildLoader.scala:223)
[error] 	at sbt.internal.Load$.loadURI$1(Load.scala:539)
[error] 	at sbt.internal.Load$.loadAll(Load.scala:555)
[error] 	at sbt.internal.Load$.loadURI(Load.scala:485)
[error] 	at sbt.internal.Load$.load(Load.scala:464)
[error] 	at sbt.internal.Load$.$anonfun$apply$1(Load.scala:253)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.apply(Load.scala:253)
[error] 	at sbt.internal.Load$.buildPluginDefinition(Load.scala:1261)
[error] 	at sbt.internal.Load$.buildPlugins(Load.scala:1191)
[error] 	at sbt.internal.Load$.plugins(Load.scala:1174)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$2(Load.scala:683)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$1(Load.scala:683)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.loadUnit(Load.scala:677)
[error] 	at sbt.internal.Load$.$anonfun$builtinLoader$4(Load.scala:477)
[error] 	at sbt.internal.BuildLoader$.$anonfun$componentLoader$5(BuildLoader.scala:158)
[error] 	at sbt.internal.BuildLoader.apply(BuildLoader.scala:223)
[error] 	at sbt.internal.Load$.loadURI$1(Load.scala:539)
[error] 	at sbt.internal.Load$.loadAll(Load.scala:555)
[error] 	at sbt.internal.Load$.loadURI(Load.scala:485)
[error] 	at sbt.internal.Load$.load(Load.scala:464)
[error] 	at sbt.internal.Load$.$anonfun$apply$1(Load.scala:253)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.apply(Load.scala:253)
[error] 	at sbt.internal.Load$.defaultLoad(Load.scala:69)
[error] 	at sbt.BuiltinCommands$.liftedTree1$1(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.doLoadProject(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.$anonfun$loadProjectImpl$2(Main.scala:680)
[error] 	at sbt.Command$.$anonfun$applyEffect$4(Command.scala:134)
[error] 	at sbt.Command$.$anonfun$applyEffect$2(Command.scala:130)
[error] 	at sbt.MainLoop$.processCommand(MainLoop.scala:153)
[error] 	at sbt.MainLoop$.$anonfun$next$2(MainLoop.scala:136)
[error] 	at sbt.State$$anon$1.runCmd$1(State.scala:242)
[error] 	at sbt.State$$anon$1.process(State.scala:248)
[error] 	at sbt.MainLoop$.$anonfun$next$1(MainLoop.scala:136)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.MainLoop$.next(MainLoop.scala:136)
[error] 	at sbt.MainLoop$.run(MainLoop.scala:129)
[error] 	at sbt.MainLoop$.$anonfun$runWithNewLog$1(MainLoop.scala:107)
[error] 	at sbt.io.Using.apply(Using.scala:22)
[error] 	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:101)
[error] 	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:57)
[error] 	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:42)
[error] 	at sbt.MainLoop$.runLogged(MainLoop.scala:34)
[error] 	at sbt.StandardMain$.runManaged(Main.scala:113)
[error] 	at sbt.xMain.run(Main.scala:76)
[error] 	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:109)
[error] 	at xsbt.boot.Launch$.withContextLoader(Launch.scala:128)
[error] 	at xsbt.boot.Launch$.run(Launch.scala:109)
[error] 	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
[error] 	at xsbt.boot.Launch$.launch(Launch.scala:117)
[error] 	at xsbt.boot.Launch$.apply(Launch.scala:18)
[error] 	at xsbt.boot.Boot$.runImpl(Boot.scala:41)
[error] 	at xsbt.boot.Boot$.main(Boot.scala:17)
[error] 	at xsbt.boot.Boot.main(Boot.scala)
[error] java.lang.NoClassDefFoundError: com/sun/jna/Platform
[error] Use 'last' for the full log.
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? 
```

--- a/plugin-tester-java/project/build.properties
+++ b/plugin-tester-java/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=1.1.1
+sbt.version=1.1.4

--- a/plugin-tester-scala/project/build.properties
+++ b/plugin-tester-scala/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=1.1.1
+sbt.version=1.1.4

--- a/project/build.properties
+++ b/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=1.1.1
+sbt.version=1.1.4

--- a/sbt-plugin/src/sbt-test/gen-java/01-gen-basic-java/project/build.properties
+++ b/sbt-plugin/src/sbt-test/gen-java/01-gen-basic-java/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=1.1.1
+sbt.version=1.1.4

--- a/sbt-plugin/src/sbt-test/gen-scala-server/00-interop/project/build.properties
+++ b/sbt-plugin/src/sbt-test/gen-scala-server/00-interop/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=1.1.1
+sbt.version=1.1.4

--- a/sbt-plugin/src/sbt-test/gen-scala-server/01-gen-basic-server/project/build.properties
+++ b/sbt-plugin/src/sbt-test/gen-scala-server/01-gen-basic-server/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=1.1.1
+sbt.version=1.1.4

--- a/sbt-plugin/src/sbt-test/gen-scala-server/02-multiple-services/project/build.properties
+++ b/sbt-plugin/src/sbt-test/gen-scala-server/02-multiple-services/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=1.0.3
+sbt.version=1.1.4

